pkg/cmd/search/shared/shared.go

```diff
@@ -114,26 +114,28 @@ func displayIssueResults(io *iostreams.IOStreams, et EntityType, results search.
 		}
 		tp.EndRow()
 	}
+
+	if len(results.Items) == 0 {
```

Comment:
This file handles `search issues` and `search prs` but it looks like corresponding changes for `search repos` is missing.